metrics/src/main/java/cucumber/metrics/regulator/module/SpeedRegulatorModule.java

```diff
@@ -0,0 +1,49 @@
+package cucumber.metrics.regulator.module;
```

Comment:
I do not understand your PR, can you rephrase what you want, please? and what files?